Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-current-contest.tsx

```diff
@@ -71,6 +73,7 @@ const CurrentContestsProvider = ({ children }: ICurrentContestsProviderProps) =>
     const [ submitContestPasswordUrlParams, setSubmitContestPasswordUrlParams ] = useState<ISubmitContestPasswordUrlParams | null>(null);
     const [ submitContestPasswordErrorMessage, setSubmitContestPasswordErrorMessage ] = useState<string | null>(null);
     const [ isPasswordValid, setIsPasswordValid ] = useState<boolean | null>(null);
+    const [ remainingTimeInMilliseconds, setRemainingTimeInMilliseconds ]=useState(defaultState.state.remainingTimeInMilliseconds);
```

Comment:
missing spaces before and after the equals sign '='
should be:
`const [ remainingTimeInMilliseconds, setRemainingTimeInMilliseconds ] = useState(defaultState.state.remainingTimeInMilliseconds);`